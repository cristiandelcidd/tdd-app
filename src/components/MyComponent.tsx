import { FC } from "react";

const MyComponent: FC = () => {
  return (
    <div>
      <h1>Learning React</h1>
      <input type="text" />
      <button>Hello World</button>
    </div>
  );
};

export default MyComponent;
