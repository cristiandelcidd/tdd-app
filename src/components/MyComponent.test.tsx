import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import MyComponent from "./MyComponent";

describe("<MyComponent />", () => {
  it("should render <MyComponent />", () => {
    const wrapper = shallow(<MyComponent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
