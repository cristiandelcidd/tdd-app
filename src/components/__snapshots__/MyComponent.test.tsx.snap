// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyComponent /> should render <MyComponent /> 1`] = `
<div>
  <input
    type="text"
  />
  <button>
    Hola Mundo
  </button>
</div>
`;
